notes/meetings.md

```diff
@@ -0,0 +1,268 @@
+# Meetings
+
+For quite some time I haven't gave much thought about
+meetings because I worked free from them for some years,
+as soon as I was exposed to them again the effect on me was
+surprising, not that I expected it to feel nice, but it stressed
+me much more than I expected (perhaps I'm just getting old) and
+that got me thinking on why it pissed me off that much.
+
+Besides the reason for stress and discomfort, I will also
+try to explain some other not as stressful inefficiencies
+of meetings and also why it is so easy for management to
+oversee these issues and also why developers sometimes "ask"
+for meetings.
+
+# Schedule Control and Interruptions
+
+This one is a more subtle source of stress, which is the
+lack of control of your own schedule. This is specially
+true for me because I have very little control over when
+I feel energetic and inspired to tackle hard problems, if
+this moment coincides with meetings this causes a great
+deal of loss of productivity and stress/anxiety. It can be even
+worse then a lack of productivity, because the ideas you would
+have in this time of bliss may not be achieved in a different
+time and you will end up with a suboptimal solution (specially
+if you need to deliver something soon, you can't wait for another
+inspired moment).
+
+The only way to avoid this is to have an average performance
+all the time, like a robot. Since humans are not robots I don't
+think anyone is capable of this regarding creative work. If your
+work is repetitive and does not require imagination/creative effort,
+then having control over your schedule to capture inspiration
+is less important, it is more easy to be interrupted and then
+go back to work.
+
+In the end "creative work" ends up being tasks
```

Comment:
Just a suggestion, rephrase it as 'So "creative work" ends up being tasks...'

Reply:
```suggestion
So "creative work" ends up being tasks
```